gobo/awesome/sound.lua

```diff
@@ -184,15 +186,19 @@ function sound.new()
          local x = mouse.screen.geometry.width - 800
          local y = 24
          local killed = false
-         for c in awful.client.iterate(function (c) return c.name == "ncpamixer" end, nil, mouse.screen) do
+         for c in awful.client.iterate(function (c) return c.name == ""..mixer.."" end, nil, mouse.screen) do
```

Comment:
```suggestion
         for c in awful.client.iterate(function (c) return c.name == mixer end, nil, mouse.screen) do
```

no need for concatenating empty strings